interface/new/new_comprehensive.php

```diff
@@ -236,14 +270,14 @@ function ins_search(ins) {
       }
 
       function checkNum () {
-       var re= new RegExp();
-       re = /^\d*\.?\d*$/;
-       str=document.forms[0].monthly_income.value;
-       if(re.exec(str))
-       {
-       }else{
-        alert("Please enter a dollar amount using only numbers and a decimal point.");
-       }
+        var re= new RegExp();
+        re = /^\d*\.?\d*$/;
+        str=document.forms[0].monthly_income.value;
+        if(re.exec(str))
+        {
+        }else{
+          alert("Please enter a dollar amount using only numbers and a decimal point.")
```

Comment:
"Monetary", not dollar.  Dollars are semi-country specific unless you wrap these in xlt() with an ADDITIONAL xlt that replaces local currency name as well as putting it in in linguistic syntax order.  Big challenge there...so monetary or currency vs. dollar.